assets/src/edit-story/components/tooltip/index.js

```diff
@@ -163,34 +119,57 @@ function WithTooltip({
   ...props
 }) {
   const [shown, setShown] = useState(false);
+  const ref = useRef(null);
+  const spacing = useMemo(
+    () => ({
+      x:
+        placement.startsWith('left') || placement.startsWith('right')
+          ? SPACING
+          : 0,
+      y:
+        placement.startsWith('top') || placement.startsWith('bottom')
+          ? SPACING
+          : 0,
+    }),
+    [placement]
+  );
 
   return (
-    <Wrapper
-      {...props}
-      onPointerEnter={(e) => {
-        setShown(true);
-        onPointerEnter(e);
-      }}
-      onPointerLeave={(e) => {
-        setShown(false);
-        onPointerLeave(e);
-      }}
-      onFocus={(e) => {
-        setShown(true);
-        onFocus(e);
-      }}
-      onBlur={(e) => {
-        setShown(false);
-        onBlur(e);
-      }}
-    >
-      <Container placement={placement}>
+    <>
+      <Wrapper
+        onPointerEnter={(e) => {
+          setShown(true);
+          onPointerEnter(e);
+        }}
+        onPointerLeave={(e) => {
+          setShown(false);
+          onPointerLeave(e);
+        }}
+        onFocus={(e) => {
+          setShown(true);
+          onFocus(e);
+        }}
+        onBlur={(e) => {
+          setShown(false);
+          onBlur(e);
+        }}
+        ref={ref}
+        {...props}
+      >
+        {children}
+      </Wrapper>
+      <Popup
+        anchor={ref}
+        placement={placement}
+        spacing={spacing}
+        isOpen={shown}
```

Comment:
Probably shouldn't be shown if the tooltip doesn't have content (title/shortcut) set.